src/components/AnswersReview.vue

```diff
@@ -22,10 +22,17 @@
     <quest-card :question="question" />
     <!-- eslint-disable-next-line vue/no-v-html -->
     <div v-html="question.answerExplanationHTML" />
+    <button
```

Comment:
Dlaczego mamy dwa przyciski z tą samą zawartością? Czy dałoby radę załatwić to jednym przyciskiem?

Reply:
no to jest taki hack, ustawienie jednego guzika na absolute sprawia, że on zwolniłby miejsce jakby w tym divie i ten guzik by zasłaniał tekst, nawet po zjechaniu na sam dół. Jak jest ten guzik-duck to robi się miejsce na ten floatujący

Reply:
Może dałoby się to jakoś zrobić CSS, albo JSem, to rozwiązanie daje pewność, że wysokość będzie odpowiadać temu guzikowi

Reply:
Myślę żę ten drugi przycisk jest konfudujący - lepiej byłoby to zmienić na np "spacer" (tak jak to @ppeczek nazwał) albo nawet dodać margin/padding to diva z contentem, chociaż nie wiem czy to drugie na 100% zadziała